app/assets/javascripts/analytics.js.erb

```diff
@@ -2,6 +2,25 @@
 var element = document.querySelector('[data-module="cookie-banner"]')
 new GOVUK.Modules.CookieBanner().start($(element))
 
+var redactPII = function() {
+  const params = new URLSearchParams(window.location.search)
```

Comment:
`URLSearchParams` is not supported in Internet Explorer (11 and below) so we can't rely on it, unfortunately.